apps/tmbitwarden.sh

```diff
@@ -78,22 +92,149 @@ cd /root
 curl_to_dir "https://raw.githubusercontent.com/bitwarden/core/master/scripts" "bitwarden.sh" "/root"
 chmod +x /root/bitwarden.sh
 check_command ./bitwarden.sh install
-sed -i "s|http_port.*|http_port: 8080|g" /root/bwdata/config.yml
-sed -i "s|https_port.*|https_port: 8443|g" /root/bwdata/config.yml
+
+# Check if all ssl settings were entered correctly
+if grep ^url /root/bwdata/config.yml | grep -q https || grep ^url /root/bwdata/config.yml | grep -q localhost
+then
+    message "It seems like you have entered some wrong settings. We will remove bitwarden now again so that you can start over again."
+    check_command docker rm bitwarden-nginx bitwarden-admin bitwarden-events bitwarden-attachments \
+    bitwarden-identity bitwarden-api bitwarden-web bitwarden-icons bitwarden-notifications bitwarden-mssql
+    rm -rf /root/bwdata
+    exit 1
+fi
+
+# Continue with the installation
+sed -i "s|http_port.*|http_port: 5178|g" /root/bwdata/config.yml
+sed -i "s|https_port.*|https_port: 5179|g" /root/bwdata/config.yml
```

Comment:
Why these ports? Are they Bitwarden default?

Reply:
no, we only cannot use prots 80 and 443 and also not the ports that onlyoffice, collabroa & co use (have to be special)
Those are the ports that the guide that I followed (dennisnotes...) has used...

Reply:
OK :+1:

Reply:
Please add that to the NONO_PORTS in the TURN_INSTALL in lib.sh.

Reply:
Hm can you guide me where I find those NONO_PORTS? a search didn't find them and it is also not in TURN_INSTALL if I looked correctly...

Reply:
ah I found them: are in talk.sh

Reply:
fixed